judge/contest_format/registry.py

```diff
@@ -0,0 +1,16 @@
+from django.utils import six
+
+formats = {}
+
+
+def register_contest_format(name):
```

Comment:
Maybe this should also proactively take a help text, describing the format. Then we could eventually show this in the admin site.